lib/views/widgets/social_button.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_ecommerce/utilities/constants/color_manager.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+class SocialButton extends StatelessWidget {
+  const SocialButton({
+    Key? key,
+    required this.iconName,
+    required this.onPress,
+  }) : super(key: key);
+  final String iconName;
+  final Function onPress;
+  @override
+  Widget build(BuildContext context) {
+    return InkWell(
+      onTap: () {
+        onPress();
+      },
+      child: Container(
+        child: Center(
+          child: SvgPicture.asset(
+            iconName,
+            width: 40.0,
+            height: 40.0,
+          ),
+        ),
+        width: 80.0,
+        height: 80.0,
+        decoration: BoxDecoration(
+          color: ColorManager.kWhite,
```

Comment:
Try to take the color from the theme, it will be better because you now refactor the code